services/director-v2/src/simcore_service_director_v2/utils/computations.py

```diff
@@ -10,48 +10,57 @@
 
 log = logging.getLogger(__file__)
 
+_COMPLETED_STATES = (RunningState.ABORTED, RunningState.FAILED, RunningState.SUCCESS)
+_RUNNING_STATES = (RunningState.STARTED, RunningState.RETRY)
+_TASK_TO_PIPELINE_CONVERSIONS = {
```

Comment:
A bit weirdly formatted.
Quesito: could you use sets as keys, since tuples take into consideration the order of the elements.

Reply:
yes, but sets are unhashable